### PR TITLE
docs: add example and docs about using Python libraries in a project

### PR DESCRIPTION
**Pull request by Angelmmiguel**
Add a new example and documentation about loading Python libraries in workers.

It refs #189

### CODE REVIEW

File: docs/docs/languages/python.md [hunk 1]
```diff
@@ -202,6 +202,83 @@ def worker(req):
 
 If you prefer, you can configure the environment variable value dynamically by following [these instructions](../features/environment-variables.md#inject-existing-environment-variables).
 
+## Python libraries
+
+The Python ecosystem has a huge number of packages. Developers like you usually rely on other libraries to accomplish different goals. In this example, you will use the [Beautiful Soup](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#quick-start) to parse a HTML document and read its text. You have all the code available in the [examples/python-libs](https://github.com/vmware-labs/wasm-workers-server/tree/main/examples/python-libs) folder.
```
Comment (ereslibre) [inline docs/docs/languages/python.md:207]:
```suggestion
The Python ecosystem has a huge number of packages. Developers like you usually rely on other libraries to accomplish different goals. In this example, you will use the [Beautiful Soup](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#quick-start) to parse an HTML document and read its text. You have all the code available in the [examples/python-libs](https://github.com/vmware-labs/wasm-workers-server/tree/main/examples/python-libs) folder.
```

File: docs/docs/languages/python.md [hunk 2]
```diff
@@ -202,6 +202,83 @@ def worker(req):
...
+
+To add a new Python library to your project, follow these steps:
+
+1. First, create an `index.py` file with the content below. This worker reads a HTML document and return the text using the `bs4` library:
```
Comment (ereslibre) [inline docs/docs/languages/python.md:211]:
```suggestion
1. First, create an `index.py` file with the content below. This worker reads an HTML document and returns the text using the `bs4` library:
```

File: docs/docs/languages/python.md [hunk 3]
```diff
@@ -202,6 +202,83 @@ def worker(req):
...
+        return res
+    ```
+
+1. Install the Beautiful Soup library with `pip` and save it in a new `_libs` folder:
+
+    ```plain
+    pip3 install -t ./_libs beautifulsoup4
+    ```
+
+1. Create a `index.toml` file with the content below. Note the name of the TOML file must match the name of the worker (`index.py` and `index.toml`). The current configuration mounts the `_libs` folder and sets the `PYTHONPATH` environment variable to the mount path:
```
Comment (ereslibre) [inline docs/docs/languages/python.md:242]:
```suggestion
1. Create an `index.toml` file with the content below. Note the name of the TOML file must match the name of the worker (`index.py` and `index.toml`). The current configuration mounts the `_libs` folder and sets the `PYTHONPATH` environment variable to the mount path:
```

File: docs/docs/languages/python.md [hunk 4]
```diff
@@ -202,6 +202,83 @@ def worker(req):
...
+        - http://127.0.0.1:8080/
+        => ./index.py
+    🚀 Start serving requests at http://127.0.0.1:8080
+    ```
+
+1. Finally, open <http://127.0.0.1:8080> in your browser.
+
+### Limitations
+
+Currently, Wasm Workers Server only supports pure Python libraries like Beautiful Soup. Libraries that requires to compile native extensions are not available yet.
```
Comment (ereslibre) [inline docs/docs/languages/python.md:280]:
```suggestion
Currently, Wasm Workers Server only supports pure Python libraries like Beautiful Soup. Libraries that requires to compile native extensions are not supported yet.
```

File: examples/python-libs/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+* Install the Python libraries
+
+  ```shell-session
+  pip3 install -r requirements.txt -t ./_libs
+  ```
+
+## Run
+
+This example runs from the previously cloned repository (See [Prerequisites](#prerequisites)). Be sure you followed all the steps and you're in the `examples/python-libs` folder:
```
Comment (ereslibre) [inline examples/python-libs/README.md:28]:
```suggestion
This example runs from the previously cloned repository (See [Prerequisites](#prerequisites)). Make sure you followed all the steps and you're in the `examples/python-libs` folder:
```

File: examples/python-libs/.wws.toml
Comment (ereslibre) [inline examples/python-libs/.wws.toml:1]: Should this file be checked in?

**Review by ereslibre — APPROVED** [review]
Great! Some minor comments from my side.

File: examples/python-libs/.wws.toml
Comment (Angelmmiguel) [reply examples/python-libs/.wws.toml:1]: Yes! It indicates `wws` to download the Python runtime when running `wws runtimes install` or passing the `--install-runtimes` / `-i` flag.

**Pull request merged**